tests/functional/object/mcg/test_s3_vector.py

```
@@ -233,3 +234,374 @@ def test_s3_vector_operations(self, vector_bucket_factory, mcg_obj):
+
+            # Step 7: Try invalid names
+            logger.info("Testing invalid bucket names")
+            invalid_names = [
+                ("special-chars-@#$", "special characters"),
+                ("CamelCaseName", "camelCase"),
+                ("UPPERCASENAME", "uppercase"),
+                ("bucket_with_underscore", "underscore"),
+            ]
+
+            for invalid_name, reason in invalid_names:
+                logger.info(f"Attempting to create bucket with {reason}: {invalid_name}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)